api/package.json

```diff
@@ -17,6 +19,7 @@
     "mongoose": "^5.6.8"
   },
   "devDependencies": {
+    "@babel/cli": "^7.5.5",
```

Comment:
It makes sense now - I was thinking that it was a native node thing that you could pre-build a node app into a format that would run better but it is done by babel. Very interesting I haven't used babel this deeply before.